academy/ibc-dev/ibcmw-integrate.md

```diff
@@ -0,0 +1,188 @@
+# Integrating IBC middleware into a chain
```

Comment:
@tmsdkeys Here the same issue applies with "acknowledgement". I changed it to American spelling for everything that is not in code tags. Please change accordingly in the code if it does not have to be British English.